backend/qemu.pm

```diff
@@ -225,6 +225,21 @@ sub start_qemu() {
     if (defined($vars->{RAIDLEVEL})) {
         $vars->{NUMDISKS} = 4;
     }
+
+    if ($vars->{NICTYPE} ne "user") {
+        # ensure MAC addresses differ globally
+        # and allow MAC addresses for more than 256 workers (up to 65535)
+        my $workerid = $vars->{WORKER_ID};
+        $vars->{NICMAC} = sprintf('52:54:00:12:%02x:%02x', int($workerid / 256), $workerid % 256);
```

Comment:
I'm not sure that's a good idea. What if we want to upload the resulting hard disk image to use it for further tests? The nic mac would change all the time then. I think for single machine jobs it's fine to use one specific mac.